src/components/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
```

Comment:
You could create another one component for this link,  to avoid code duplication.

Reply:
agree